Applications/Spire/Source/Ui/ResponsiveLabel.cpp

```diff
@@ -101,7 +97,10 @@ void ResponsiveLabel::set_current(const optional<int> mapped_label_index) {
 void ResponsiveLabel::sort_mapped_labels() {
   std::sort(m_mapped_labels.begin(), m_mapped_labels.end(),
     [=] (const auto& first, const auto& second) {
-      return first.m_pixel_width < second.m_pixel_width;
+      if(first.m_pixel_width != second.m_pixel_width) {
+        return first.m_pixel_width < second.m_pixel_width;
+      }
+      return first.m_list_index < second.m_list_index;
```

Comment:
I think `<` should be `>`, because the list index should be sorted in descending order and the lower list index should be placed in the back of the `m_mapped_labels`.

Reply:
They're sorted in ascending order, so if indices 2, 7, and 5 are the same pixel width, then they're sorted as 2, 5, 7, and the item at index 2 would be displayed.